biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -430,11 +428,12 @@ subroutine disturbance_rates( site_in, bc_in)
                (currentPatch%area - currentPatch%total_canopy_area) * harvest_rate / currentPatch%area
        endif
 
-       ! For nocomp mode, we need to prevent producing too small patches, which may produce small patches
-       if ((hlm_use_nocomp .eq. itrue) .and. &
-           (currentPatch%disturbance_rates(dtype_ilog)*currentPatch%area .lt. min_patch_area_forced)) then
-          currentPatch%disturbance_rates(dtype_ilog) = 0._r8
-       end if
+       ! cdk. Oct. 24, 2024. I don't think we need this anymore. commenting out.
+       ! ! For nocomp mode, we need to prevent producing too small patches, which may produce small patches
+       ! if ((hlm_use_nocomp .eq. itrue) .and. &
+       !     (currentPatch%disturbance_rates(dtype_ilog)*currentPatch%area .lt. min_patch_area_forced)) then
+       !    currentPatch%disturbance_rates(dtype_ilog) = 0._r8
+       ! end if
```

Comment:
@ckoven do you want to leave this commented out in case it crops up in case runs after we integrate this PR?